### DISCUSSION TITLE
Granary capacity 3500?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#664] [Q&A]
We're there any changes since 3.1.0.514 regarding granary capacity?

I had this situation:
[8Valentia1c.zip](https://github.com/Keriew/augustus/files/9464703/8Valentia1c.zip)
[8Valentia1j.zip](https://github.com/Keriew/augustus/files/9464705/8Valentia1j.zip)
![Screenshot_20220831-230758060](https://user-images.githubusercontent.com/93785920/187772657-e12134e7-cd17-484e-bd5e-f5f70965ee3e.jpg)

I don't mind the +300 capacity that much, but I got the impression that market ladies also don't like going to that granary much.

**Comment by crudelios** [comment]
That definitely shouldn't happen.

To the issue list it goes!
